src/Datadog.Trace.Tools.Runner/Datadog.Trace.Tools.Runner.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>1.19.3</Version>
+    <Title>Datadog APM - DD Autoinstrumentation Runner</Title>
+    <Copyright>Copyright 2020 Datadog, Inc.</Copyright>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp3.0;netcoreapp2.2;netcoreapp2.1;</TargetFrameworks>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>ddtrace</ToolCommandName>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <NoWarn>NU5100</NoWarn>
+    <PackageId>ddtrace</PackageId>
```

Comment:
I know Python uses `ddtrace`, but following the dotnet tool conventions is more important than consistency with other Datadog tracers. It seems that most tools follow one of several conventions:
- `dotnet-dump`, `dotnet-trace`, `dotnet-ef`, `dotnet-user-secrets`, etc
- `{Company}.{Product}.{Cli|Tool[s]}`
  - `Cake.Tool`
  - `Amazon.Lambda.Tools`
  - `GitVersion.Tool`
  - `Octopus.DotNet.Cli`
  - `SmartCode.CLI`

I suggest we use `dd-trace` because smashing words together with no punctuation to separate them is not cool 😉

Reply:
Changed both `PackageId` and `ToolCommandName` to `dd-trace` in 837c7fa

Reply:
Thanks!